projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -38,10 +38,11 @@ class Search_Widget extends \WP_Widget {
 	 */
 	const DEFAULT_SORT = 'relevance_desc';
 	/**
-	 * The Jetpack_Search instance.
+	 * Never used.
```

Comment:
Is it not used here?
https://github.com/Automattic/jetpack/blob/9d06de7b232ee5a227ce8eaf5a5bf704d78340b8/projects/packages/search/src/initializers/class-initializer.php#L207

...via here?
https://github.com/Automattic/jetpack/blob/9d06de7b232ee5a227ce8eaf5a5bf704d78340b8/projects/packages/search/src/initializers/class-initializer.php#L112